text/0002-bundler-2.md

```diff
@@ -0,0 +1,44 @@
+# Bundler 2 RFC
+
+This document outlines the changes proposed for the Bundler v2 release. The main goal of this release will be to remove
+support for Ruby < 2.0, remove and deprecate existing functionality and introduce any changes to existing behaviour.
+
+## Removed Functionality
+* The `bundle viz` and `bundle show` commands. The `viz` command will be moved to a Gem/Bundler Plugin and `show` will be removed in favour of `bundle info` and `bundle list`
```

Comment:
`bundle show` *is* removed in 2.0, it's being replaced by `bundle info`,  `bundle list` will no longer be an alias of `bundle show`. Instead it will be it's own command that only just lists the gems in the bundle the way `bundle show` currently does, it will also have an `ls` alias.

See https://github.com/bundler/bundler/issues/4754 for more info